pms/models/pms_automated_mails.py

```diff
@@ -0,0 +1,172 @@
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class PmsAutomatedMails(models.Model):
+    _name = "pms.automated.mails"
+    _description = "Automatic Mails"
+
+    name = fields.Char(string="Name")
+
+    pms_property_id = fields.Many2one(string="Property", comodel_name="pms.property")
```

Comment:
Why not company_ids? I think that the same automated mail can be used by multi  properties, what do you think?